Source/Core/Core/Movie.cpp

```diff
@@ -463,10 +474,10 @@ void ChangeWiiPads(bool instantly)
 
   for (int i = 0; i < MAX_WIIMOTES; ++i)
     if (WiimoteCommon::GetSource(i) != WiimoteSource::None)
-      controllers |= (1 << i);
+      controllers |= (1 << (i + 4));
```

Comment:
I know this isn't originally your code, but would you mind adding curlies here around the `for`? That would make it more obvious which statements belong to the loop.